lib/api/dsl/methods.js

```diff
@@ -597,7 +789,9 @@ module.exports = methods = {
 
     deferred.reject(new KuzzleError('regexp is not implemented yet.'));
     return deferred.promise;
-  }
+  },
+
+  _geoUtil: geoUtil // in order to allow tests
```

Comment:
Use ``rewire`` to test "private" methods or variables inside a module, instead of exposing them needlessly.